tests/python/quantization/test_quantization.py

```diff
@@ -270,7 +270,7 @@ def check_quantized_pooling(data_shape, kernel, pool_type, pad, stride, global_p
 def test_quantized_fc():
     def check_quantized_fc(data_shape, num_hidden, no_bias, qdtype, flatten=True):
         if mx.current_context().device_type != 'gpu':
```

Comment:
We should be able to run this test on CPU in CI.  Could we test to see if 'MKL' is in the env var 'BUILD_TAG' and run the test if it is.

Reply:
@KellenSunderland good suggestion! Currently, the CI doesn't include Intel MKL library as BLAS library and @azai91 is working on adding it so that we can have a better coverage, such as batch_gemm, quantization FC, etc.

Reply:
fixed

Reply:
@pengzhao-intel Oh sorry, didn't realize that was the case.   If the tests won't pass without full mkl installed and it's not there let's add this in a later PR.

Reply:
@pengzhao-intel do you mean the full MKL? We already use [MKLML on CI](https://github.com/apache/incubator-mxnet/blob/master/ci/docker/install/ubuntu_mklml.sh).

Reply:
@lebeg yes, I mean full MKL. The MKLML doesn't have the INT8 GEMM now :)